docs/topics/settings.rst

```diff
@@ -1620,6 +1620,7 @@ Scope: ``spidermiddlewares.urllength``
 
 The maximum URL length to allow for crawled URLs. For more information about
 the default value for this setting see: https://support.microsoft.com/en-us/topic/maximum-url-length-is-2-083-characters-in-internet-explorer-174e7c8a-6666-f4e0-6fd6-908b53c12246
+If URLLENGTH_LIMIT is equals to ``0``, then the :class:`UrlLengthMiddleware` middleware is disabled.
```

Comment:
There should be an empty line above to make this new line a separate paragraph, otherwise it is appended to the end of the text above.

There’s also a minor grammatical number discordance (“equals” should be “equal”), but you can actually just remove “equals to” altogether and the sentence will have the same meaning with fewer words, which I personally tend to favor :)

Finally, you might want to render the documentation locally (`tox -e docs`) and check if that class link works. You may need to replace it with the full import path, which you can prepend with a tilde so that the display text is only the class name: ``` :class:`scrapy.spidermiddlewares.urllength.UrlLengthMiddleware` ```.